engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -1769,7 +1790,9 @@ object Runtime {
     private lazy val mapper = {
       val factory = new CBORFactory()
       val mapper  = new ObjectMapper(factory) with ClassTagExtensions
-      mapper.registerModule(DefaultScalaModule)
+      mapper
+        .registerModule(DefaultScalaModule)
+        .registerModule(ScalaObjectDeserializerModule)
```

Comment:
I wonder, why it is a separate module and not a part of the Scala one.

Reply:
Yeah, clearly someone messed that mixed that up